admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java

```diff
@@ -137,15 +137,28 @@ protected void validateFileDescriptions() {
     @Override
     public String joinCategory() {
         fetchFileDescriptions();
+        fetchMetadataEdit();
         return super.joinCategory();
     }
 
     @Override
     public String removeCategory() {
         fetchFileDescriptions();
+        fetchMetadataEdit();
         return super.removeCategory();
     }
 
+    protected void fetchMetadataEdit() {
+        if (ApsAdminSystemConstants.EDIT == this.getStrutsAction()) {
+            try {
+                this.setMetadata(this.loadResource(this.getResourceId()).getMetadata());
+            } catch (Throwable ex) {
+                logger.debug("error reading resource on fetchMetadataEdit {}", ex);
+            }
+            logger.debug("metadata size: {}", getMetadata().size());
```

Comment:
Would it be possible to add some identifying issue to this message so we know which asset it applies to?